scripts/update-registry/update2.ts

```diff
@@ -49,6 +51,19 @@ interface DescribeResult {
   updateProjectScratchDef(missingTypes);
   // TODO: sourceApi has to match the coverage report
   if (!process.env.RB_EXISTING_ORG) {
+    const hasDefaultDevHub = Boolean(
+      JSON.parse(shelljs.exec('sfdx config:get defaultdevhubusername --json', { silent: true }).stdout).result[0].value
```

Comment:
Check that a default devhub is available before trying to create a scratch org.
Error msg shows how to set a default hub with `sfdx`.